3rdparty/everest/library/Hacl_Curve25519_joined.c

```diff
@@ -18,6 +18,12 @@
  *
  *  This file is part of mbed TLS (https://tls.mbed.org)
  */
+#ifndef _BSD_SOURCE
+#define _BSD_SOURCE /* Required to enable compilation under --std=c99 */
```

Comment:
So actually I think this should be

```suggestion
/* Required to get htole64() from endian.h */
#define _BSD_SOURCE
```